tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -980,7 +1007,7 @@ def register(
         fixed_mask = np.uint8(fixed_mask > 0)
         moving_mask = np.uint8(moving_mask > 0)
 
-        # Perform Pre-alignment
+        # -- Perform Pre-alignment --
```

Comment:
I don't have a problem with this comment formatting, but we don't do it anywhere else in the repo. It would be good to try and keep commenting style consistent. What do you think @shaneahmed?

Reply:
I had them differently to differentiate between different steps of registration. I have now modified it and other comments as well.
https://github.com/TissueImageAnalytics/tiatoolbox/blob/a27d716ef2e9742c292f1b76a4981ca44d971742/tiatoolbox/tools/registration/wsi_registration.py#L1010-L1011